### PR TITLE
Add semantic release tool integration.

### PR DESCRIPTION
**Pull request by grianbrcom**
Signed-off-by: Anton Grigorev <anton.grigorev@broadcom.com>

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/releaseScripts/prepare.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```
Comment (ishche) [inline clients/cobol-lsp-vscode-extension/releaseScripts/prepare.sh]: Copyright header is messing

File: clients/cobol-lsp-vscode-extension/package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     },
     "repository": {
         "type": "git",
-        "url": "https://github.com/eclipse/che-che4z-lsp-for-cobol.git"
+        "url": "git@github.com:grianbrcom/che-che4z-lsp-for-cobol.git"
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/package.json]: Why do you use ssh here?

File: clients/cobol-lsp-vscode-extension/releaseScripts/prepare.sh (hunk 1 above)
Comment (grianbrcom) [reply clients/cobol-lsp-vscode-extension/releaseScripts/prepare.sh]: You are right. I'll add it to here.

File: clients/cobol-lsp-vscode-extension/package.json (hunk 2 above)
Comment (grianbrcom) [reply clients/cobol-lsp-vscode-extension/package.json]: Good catch. I'm used to using ssh. I'll check the work with https.

Comment (grianbrcom) [reply clients/cobol-lsp-vscode-extension/package.json]: The plugin works well with https.

**Comment by grianbrcom** [comment]
@temanbrcom, now it could parse our existing tags.

**Comment by temanbrcom** [comment]
CQ for `semantic-release` is still in progress: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21742

**Comment by zacanbrcom** [comment]
@grianbrcom don't forget to include your contribution in the changelog!! :D

**Comment by grianbrcom** [comment]
Thanks, @zacanbrcom, I've fixed it.

**Pull request merged**